resources/quality/kemiq_q2/kemiq_q2_beta_abs_draft.inst.cfg

```diff
@@ -0,0 +1,42 @@
+[general]
+version = 2
+name = Draft
+definition = kemiq_q2_beta
+
+[metadata]
+type = quality
+weight = -3
+material = generic_abs
+quality_type = coarse
+setting_version = 1
```

Comment:
We are now using `setting_version = 2`, please update this for all the quality files.